.github/workflows/windows_unit_tests.yml

```diff
@@ -38,12 +40,29 @@ jobs:
           python -c "import neuron"
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
       - name: Install dependencies and HNN
         shell: cmd
         run: |
           python -m pip install --upgrade pip
           pip install --verbose .[opt,parallel,test,gui]
-      - name: Test with pytest
+
+      - name: Check ruff formatting
+        shell: cmd
+        run: |
+          python -m ruff format hnn_core --check
+
+      - name: Lint with ruff
+        shell: cmd
+        run: |
+          python -m ruff check hnn_core
+
+      - name: Test non-MPI, embarrassingly parallel tests with pytest
```

Comment:
This is great! Were embarrassingly parallel tests blocked previously? Or is it just MPI since it's now pip installable on windows if I remember our previous conversation correctly